src/addElasticsearchNode.ts

```diff
@@ -1,6 +1,61 @@
-export async function handler(event: ClusterStatusResponse): Promise<AddElasticsearchNodeResponse> {
+import {ssmCommand} from './utils/ssmCommand';
+import {increaseAsgSize, describeAsg} from './utils/autoscaling';
+import {StandardOutputContent} from 'aws-sdk/clients/ssm';
+import {AutoScalingGroupsType} from "aws-sdk/clients/autoscaling";
+
+export async function handler(event): Promise<AddElasticsearchNodeResponse> {
```

Comment:
can we declare the type of `event` with something from `handlerResponses.ts` now?